offline/upgrading.md

```diff
@@ -165,27 +192,32 @@ restund_uid = root
 ```
 
 ### Deeplink
+
 If you are using the old deeplink process (deprecated!), set:
```

Comment:
This has been deprecated for many years; file a ticket to remove minio deeplink support completely.